library/general/disallowedtags/suite.yaml

```diff
@@ -23,10 +23,10 @@ tests:
     object: samples/container-image-must-not-have-latest-tag/example_disallowed_tag.yaml
     assertions:
     - violations: yes
-  - name: single-disallowed-tag-ephemeral
-    object: samples/container-image-must-not-have-latest-tag/example_disallowed_tag_ephemeral.yaml
-    assertions:
-    - violations: yes
+  # - name: single-disallowed-tag-ephemeral
```

Comment:
Do we need to disable the suite tests? they should be unaffected by the issues live k8s tests have